ExadelTimeTrackingSystem/ExadelTimeTrackingSystem.BusinessLogic/Extensions/ObjectExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace ExadelTimeTrackingSystem.BusinessLogic.Extensions
+{
+    using Newtonsoft.Json;
+
+    public static class ObjectExtensions
+    {
+        static ObjectExtensions()
```

Comment:
don't need it

Reply:
empty constructor I mean, it's called default constructor, and it will be created automatically, you don't need to initialize it